CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## 30 Release Notes
 
+### Breaking Changes
+
+* All endpoints now require valid version numbers.
+
+* `POST /api/workflows/v2` now requires a `workflowType` and a `workflowTypeVersion`. These are optional for the `v1`
+api. See the [README](https://github.com/broadinstitute/cromwell#post-apiworkflowsversion) for more information.
```

Comment:
ToL does this mean there is an official V2 of all the endpoints, or just `/api/workflows`?

Reply:
Metadata v2 already existed (it's the _exact_ same as v1 due to... at this point no good reason). POST /api/workflows, aka workflow submissions, will now have a v2 that behave differently.

Reply:
Huh, TIL!

Reply:
Perhaps this should be a TT tomorrow after standup but I don't think we should be having multiple versions of every endpoint. That's not how most sites do their APIs, at least not in my experience. Every endpoint should accept a `v2` parameter, IMO.